docs/advanced/additional-requests.rst

```diff
@@ -543,9 +544,9 @@ The key takeaways for this example are:
 
 .. tip::
 
-    The :meth:`~.HttpClient.batch_execute` method can accept different varieties
-    of :class:`~.HttpRequest` that MAY NOT be related with one another. For
-    example, it could be a mixture of ``GET`` and ``POST`` requests or even
+    The :meth:`~.HttpClient.batch_execute` method can multiple
+    :class:`~.HttpRequest` instances. For example, it could be a mixture
+    of ``GET`` and ``POST`` requests or even
```

Comment:
```suggestion
    The :meth:`~.HttpClient.batch_execute` method can execute multiple
    :class:`~.HttpRequest` instances. For example, it could be a mixture
    of ``GET`` and ``POST`` requests or even
```